.ci/lint_r_code.R

```diff
@@ -0,0 +1,82 @@
+
+library(lintr)
+
+args <- commandArgs(
+    trailingOnly = TRUE
+)
+SOURCE_DIR <- args[[1]]
+
+FILES_TO_LINT <- list.files(
+    path = SOURCE_DIR
+    , pattern = "\\.r$"
+    , all.files = TRUE
+    , ignore.case = TRUE
+    , full.names = TRUE
+    , recursive = TRUE
+    , include.dirs = FALSE
+)
+
+# Some linters from the lintr package have not made it to CRAN yet
```

Comment:
How many (approximately: 1, 2, or more) linters are not available with the package version installed from CRAN?

Reply:
it's on the order of 4-6. Yeah, I kind of wanted to test how the installation from source plays with `conda` R, and the only way I knew to reliably reproduce the environment on Travis was to just directly try on Travis.

I'll take out the unreleased ones for now and create an issue (closed and attached to #2302 ) for adding the new ones once `lintr` gets updated. I'm also going to gently open a "can you please do a release" issue on `lintr`, since it has been almost a year since the last one.

Reply:
Thank you very much! I think it's the most appropriate way to introduce the linting and do not suffer from it!

Reply:
I have added [a comment](https://github.com/jimhester/lintr/issues/318#issuecomment-536313549) on jimhester/lintr#318 asking for a new release.